integration/rootless/rootless_test.sh

```diff
@@ -27,19 +28,40 @@ setup() {
 
 cleanup() {
 	sudo sed -i -e 's/^.*\(rootless\)/# \1/g' /usr/share/defaults/kata-containers/configuration.toml
-	sudo crictl stopp "$pod_id" || true
-	sudo crictl rmp "$pod_id" || true
+	sudo crictl stopp "$pod_id" &>/dev/null || true
+	sudo crictl rmp "$pod_id" &>/dev/null || true
 }
 
 run() {
 	pod_id="$(sudo crictl runp -r kata "${dir_path}/rootless-pod.json")"
 	waitForProcess 15 3 "sudo crictl inspectp "$pod_id" | jq '.status.state' | grep 'SANDBOX_READY'"
+
+	# There must be created one and only one temporary user.
+	local grep_uid="grep -e '^kata-[0-9]\+' /etc/passwd"
+	[ "$(eval $grep_uid | wc -l)" -eq 1 ] || \
+		die "Unexpected more than one kata temporary UID"
+	local kata_uid="$(eval $grep_uid | cut -d: -f3)"
+	local kata_gid="$(eval $grep_uid | cut -d: -f4)"
+	[[ -n "$kata_uid" && -n "$kata_gid" ]] || \
+		die "Unabled to find the kata temporary UID"
+
+	# Assert the QEMU process is owned by that user.
+	ps -u "$kata_uid" -o cmd --no-header | \
+		grep -q "^${HYPERVISOR_PATH:-qemu}" || \
```

Comment:
If the QEMU path isn't present in the config file, `kata-runtime kata-env` falls back on the hard-coded default value `qemu-system-XXXX`  => `HYPERVISOR_PATH` is always set and non-null AFAICT.

Reply:
I was worried about a bug on `extract_kata_env` that would export a empty string.